tests/integration-tests/test-suites/helloWorld.spec.ts

```diff
@@ -0,0 +1,6 @@
+import { sumFunc } from "../utils/helpers"
+
+test('adds 1 + 2 to equal 3', () => {
+    const answer = sumFunc(1, 2)
+    expect(answer).toEqual(3)
+  })
```

Comment:
Same comment RE newline here. Although in this case you can just run `yarn lint:fix` and it should pick it up. Note, that won't work for the Dockerfile though so you still need to address that comment :)

Side note. Have you installed the pre-commit hooks? This should run linting when committing and make these sort of fixes for you.

If you haven't you can do so by running `yarn husky install`.